app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+    source_file = parts[1]
+    destination_file = parts[2]
```

Comment:
It's better to use `try` for the whole function, to catch some `ValueError` exceptions, e.g. if the user is trying to copy the file to itself. or if the input is invalid or isn't splittable to 3 elements (f.e. "cp smth_smth"), you need to raise an appropriate exception, then catch them in `except`, and print the error message.